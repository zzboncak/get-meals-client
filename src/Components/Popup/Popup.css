.popup {
    width: 200px;
    background-color: gray;
    color: white;
}