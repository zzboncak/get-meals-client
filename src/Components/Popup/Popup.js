import React from 'react';
import './Popup.css';

class Popup extends React.Component {
    render() {
        return (
            <div className="popup">
                <h2 className="popup__location-title">{this.props.locationName}</h2>
                <p className="popus__location-address">{this.props.address}</p>
                <p className="popus__location-description">{this.props.description}</p>
                <p className="popus__location-hours">{this.props.hours}</p>
            </div>
        )
    }
}

export default Popup;