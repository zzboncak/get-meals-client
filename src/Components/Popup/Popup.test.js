import React from 'react';
import ReactDOM from 'react-dom';
import Popup from './Popup';

describe(`Popup component`, () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<Popup />, div);
      ReactDOM.unmountComponentAtNode(div);
  });
})
