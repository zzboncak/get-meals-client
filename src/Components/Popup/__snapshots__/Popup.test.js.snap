// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup component renders the component as expected 1`] = `
<div
  className="popup"
>
  <h2
    className="popup__location-title"
  />
  <p
    className="popus__location-address"
  />
  <p
    className="popus__location-description"
  />
  <p
    className="popus__location-hours"
  />
</div>
`;
